lib/ood_core/job/adapters/helper.rb

```diff
@@ -12,6 +12,17 @@ class Helper
         def self.bin_path(cmd, bin_default, bin_overrides)
           bin_overrides.fetch(cmd.to_s) { Pathname.new(bin_default.to_s).join(cmd.to_s).to_s }
         end
+        
+        # Gets a command that submits command on another host via ssh
+        # @param cmd [String] the desired command to execute on another host
+        # @param submit_host [String] where to submit the command
+        # @return [String] command wrapped in ssh if submit_host is present
+        def self.ssh_wrap(cmd, submit_host)
+          if submit_host.empty?
+            return cmd
+          end
+          "ssh #{submit_host} \"#{cmd}\""
```

Comment:
Check the implementation of this for the Linux host adapter: https://github.com/OSC/ood_core/blob/63187f8874e62b752a73e39263f539ce0241495a/lib/ood_core/job/adapters/linux_host/launcher.rb#L119-L145

In particular the known hosts issue and the `'-o', 'BatchMode=yes'` argument.

Note I'm going to open a another issue to see if we can handle known_hosts in a better way.